src/utils/ConfigHelper.ts

```diff
@@ -141,6 +141,57 @@ const configs: ConfigHelperConfig[] = [
     dispenserAddress: null,
     metadataContractAddress: null,
     startBlock: 90707
+  },
+  {
+    networkId: 2021000,
+    network: 'gaiaxtestnet',
+    nodeUri: 'https://gaia-x.rpc',
+    metadataCacheUri: 'https://aquarius.gaiaxtestnet.oceanprotocol.com',
+    providerUri: 'https://provider.gaiaxtestnet.oceanprotocol.com',
+    subgraphUri: 'https://subgraph.gaiaxtestnet.oceanprotocol.com',
+    explorerUri: 'https://gaiaxtestnet.explorer',
+    oceanTokenAddress: null,
+    oceanTokenSymbol: 'OCEAN',
+    factoryAddress: null,
+    poolFactoryAddress: null,
+    fixedRateExchangeAddress: null,
+    dispenserAddress: null,
+    metadataContractAddress: null,
+    startBlock: 0
+  },
+  {
+    networkId: 80001,
+    network: 'mumbai',
+    nodeUri: 'https://polygon-mumbai.infura.io/v3/',
```

Comment:
so could we switch to infura for polygon mainnet too? Either way in market we have to adapt our method for dynamically adding the infura ID which so far we can only do hard coded as there is no real pattern here: https://github.com/oceanprotocol/market/blob/main/src/utils/ocean.ts#L14-L26